locales/it/index.yml

```
@@ -54,7 +54,7 @@ global:
   localNotifications:
     spidLogin:
       title: Ricordati di accedere ad IO!
-      message: Accedi con la tua identità SPID su IO, l’app dei servizi pubblici
+      message: Accedi con il tuo profilo digitale su IO, l’app dei servizi pubblici
```

```suggestion
      message: Accedi con la tua identità digitale su IO, l’app dei servizi pubblici
```
Sorry, my bad.
This could be more appropriate

---

Ok no problem!
Maybe even the English one should be changed to "digital identity" ?
If that's ok i may change it and add the commit to the PR

---

Yes! The same for english too ;)